manifests/base/resources/qlikview-client/transformers/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  app: qlikview-client
+resources:
+- ../../../transformers
+transformers:
+- ../../../../../.operator/transformers
+- ../../../../../configuration/transformers
+#Disable all transformers except these
+patches:
+  - target:
+      labelSelector: >-
+        key notin (
+          ingressClass,
```

Comment:
dont need ingressAuthUrl ??

Reply:
Not sure about ingress auth...
`
MacBook-Pro:qlikview-client dlx$ pwd
/tmp/.chartcache/qliksense/charts/qlikview-client
MacBook-Pro:qlikview-client dlx$ helm template .
---
# Source: qlikview-client/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: release-name-qlikview-client
  labels:
    app: qlikview-client
    chart: qlikview-client-1.0.12
    release: release-name
    heritage: Tiller
  annotations:
    prometheus.io/scrape: "true"
    prometheus.io/port: "9180"
spec:
  type: ClusterIP
  ports:
    - port: 9003
      targetPort: 9003
      protocol: TCP
      name: qlikview-client
  selector:
    app: qlikview-client
    release: release-name

---
# Source: qlikview-client/templates/deployment.yaml
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: release-name-qlikview-client
  labels:
    app: qlikview-client
    chart: qlikview-client-1.0.12
    release: release-name
    heritage: Tiller
spec:
  replicas: 1
  template:
    metadata:
      labels:
        app: qlikview-client
        release: release-name
    spec:
      containers:
        - name: qlikview-client
          image: qlik-docker-qsefe.bintray.io/qlikview-client:12.50.2055
          imagePullPolicy:
          ports:
            - containerPort: 9003
          livenessProbe:
            httpGet:
              path: /health
              port: 9003
      imagePullSecrets:
        - name: artifactory-docker-secret

  selector:
    matchLabels:
      app: qlikview-client
      release: release-name

---
# Source: qlikview-client/templates/ingress.yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: release-name-qlikview-client
  labels:
    app: qlikview-client
    chart: qlikview-client-1.0.12
    release: release-name
    heritage: Tiller
  annotations:
    nginx.ingress.kubernetes.io/rewrite-target: "/opendoc.htm"
    kubernetes.io/ingress.class: nginx
spec:
  rules:
    - http:
        paths:
          - path: /qv/opendoc.htm
            backend:
              serviceName: release-name-qlikview-client
              servicePort: 9003
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: release-name-qlikview-client-ajax
  labels:
    app: qlikview-client
    chart: qlikview-client-1.0.12
    release: release-name
    heritage: Tiller
  annotations:
    kubernetes.io/ingress.class: nginx
spec:
  rules:
    - http:
        paths:
          - path: /QvAjaxZfc
            backend:
              serviceName: release-name-qlikview-client
              servicePort: 9003

MacBook-Pro:qlikview-client dlx$
`

Reply:
Do these 2 ingresses look unauthenticated to you?

Reply:
@ffoysal Actually, this is what the 2 ingresses look like after our "kustomizations":
```
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: qlik-nginx
    nginx.ingress.kubernetes.io/auth-url: http://qliksense-edge-auth.default.svc.cluster.local:8080/v1/auth
    nginx.ingress.kubernetes.io/rewrite-target: /opendoc.htm
  labels:
    app: qlikview-client
    chart: qlikview-client-1.0.12
    heritage: Tiller
    release: qliksense
  name: qliksense-qlikview-client
  namespace: default
spec:
  rules:
  - http:
      paths:
      - backend:
          serviceName: qliksense-qlikview-client
          servicePort: 9003
        path: /qv/opendoc.htm
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: qlik-nginx
    nginx.ingress.kubernetes.io/auth-url: http://qliksense-edge-auth.default.svc.cluster.local:8080/v1/auth
  labels:
    app: qlikview-client
    chart: qlikview-client-1.0.12
    heritage: Tiller
    release: qliksense
  name: qliksense-qlikview-client-ajax
  namespace: default
spec:
  rules:
  - http:
      paths:
      - backend:
          serviceName: qliksense-qlikview-client
          servicePort: 9003
        path: /QvAjaxZfc
---
```

Reply:
@ffoysal What does the `ingressAuthUrl` do?

Reply:
@ffoysal I just tried inserting the `ingressAuthUrl` transformer and re-running `kuz` and it gave me the same output as without it...